pex/pep425.py

```diff
@@ -99,6 +99,14 @@ def _iter_supported_tags(cls, impl, version, platform):
     """
     # Predict soabi for reasonable interpreters.  This is technically wrong but essentially right.
     abis = []
+    if impl == 'cp' and version.startswith('2'):
```

Comment:
I did that initially, but wasn't sure if all the abi tags from cp3 also applied to cp2; the docs on seems to be lacking on this topic. The 2 separate conditionals allows for keeping the 2 lists independent.

I can extract out the common ones if that's preferred. Let me know.